embedding/cpp/elm_flutter_view.cc

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "include/elm_flutter_view.h"
+
+#include "tizen_log.h"
+#include "utils.h"
+
+bool ElmFlutterView::OnCreate() {
```

Comment:
How about changing the method name? `OnCreate` should be used for the event.

Reply:
I changed the method name as below.
`bool RunFlutterEngine(void *elm_parent);`
`bool RunFlutterEngine(void *elm_parent, int32_t width, int32_t height);`